packages/components/menu/src/menu-option-group.tsx

```diff
@@ -96,7 +96,7 @@ export const MenuOptionGroup = forwardRef(
     return (
       <MenuGroup
         ref={ref}
-        className={cx("ui-menu__item--group--option", className)}
+        className={cx("ui-menu__item-group--option", className)}
```

Comment:
I might be wrong, but the `ui-menu__item-group--option` class is inconsistent with `ui-menu__group` in `packages/components/menu/src/menu-group.tsx`.

Should it be

```suggestion
        className={cx("ui-menu__group--option", className)}
```

or should the class in `packages/components/menu/src/menu-group.tsx` be not `ui-menu__group` but `ui-menu__item-group`?

Reply:
You are right.
I fixed it.